roads.mss

```diff
@@ -1182,7 +1309,14 @@ tertiary is rendered from z10 and is not included in osm_planet_roads. */
       }
     }
 
+    [feature = 'highway_road'],
     [feature = 'highway_service'] {
+      [zoom >= 13][feature = 'highway_road'] {
+        .roads-fill, .bridges-fill {
+          line-color: @road-fill;
+        }
+      }
+      [zoom >= 10][feature = 'highway_road'],
       [zoom >= 13][service = 'INT-normal'] {
         line-width: 1;
         line-color: @residential-casing;
```

Comment:
I was thinking of the color (I know it's not a problem introduced by this PR, but still cleaning it up would be nice).